Update readme to highlight imports (#228)
pointfreeco/swift-navigation: README.md

active at a time. However, SwiftUI does not come with the tools to drive navigation from this 
model. This is where the SwiftUINavigation tools becomes useful.

> > [!IMPORTANT]
> > To get access to the tools described below you must depend on the [SwiftNavigation package](https://github.com/pointfreeco/swift-navigation#installation) and
> > import the **SwiftUINavigation** library.

Reviewer: ```suggestion
> [!IMPORTANT]
> To get access to the tools described below you must depend on the SwiftNavigation package
> (see [Installation](#installation)) and import the **SwiftUINavigation** library.
```

the `destination` property:

```swift
import SwiftUINavigation 

> 

Reviewer: ```suggestion
// ...
```

powerful to be able to drive navigation from state. It allows you to encapsulate more of your 
feature's logic in an isolated and testable domain, and it unlocks deep linking for free since one
just needs to construct a piece of state that represents where you want to navigate to, hand it to
SwiftUI, and let it handle the rest.

The UIKitNavigation library brings a powerful suite of navigation tools to UIKit that are heavily
inspired by SwiftUI. For example, if you have a feature model like the one discussed above in
the [SwiftUI](#swiftui) section:

```swift
import SwiftNavigation 

@Observable
class FeatureModel {
  var destination: Destination?

> +@CasePathable

Reviewer: ```suggestion
  @CasePathable
```

the [SwiftUI](#swiftui) section:

```swift

> import SwiftUINavigation 
> 

Reviewer: Should this be `SwiftNavigation`? And wanna add the `@CasePathable` to the `Destination` enum?

```suggestion
import SwiftNavigation 

```

Author: You are right. Minimum required is SwiftNavigation
